### ISSUE TITLE
Allow ranges of logs to be moved into a note

### ISSUE THREAD

**Issue by scottbenton** [scottbenton/Iron-Journal#394]

**Comment by nickarrow-temp** [comment]
I really like inserting the roll results in-line in a journal. I am using the “copy” button to do so. I’d love a “Save to note” button that does it automatically. I don’t actually want every roll in the journal, so a button is ideal. This way, for any rolls that I want inserted in the journal, I click the button and they are inserted. 

Also, FWIW, the copy functionality formats the text as such:


Strike (edge)

Action: 1 + 3 = 4

Challenge: 4, 3

**WEAK HIT**

I’d love a more compact (and maybe even Quoted version) because I think it looks nicer in-line in a journal:

> **STRIKE:** **Action:** 1 + 3 (edge) = 4 **Challenge:** 4, 3 
> **WEAK HIT**

**Comment by scottbenton** [comment]
I agree - I'm thinking you should be able to mark a note as your current session, and then either manually choose to insert rolls or be able to set a filter to insert only certain types (excluding oracles for example). I've been pretty inspired by the work the Obsidian plugin team has been doing, so I think it will likely be a card with a quick view or a detailed view.

My own spin on this: 
![image](https://github.com/scottbenton/Iron-Fellowship_and_Crew-Link/assets/53715213/ad841d1d-7d64-4b1c-8bfc-80ddeea50ab3)
